migrations/mystery_items.js

```diff
@@ -27,6 +27,14 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
-}
-
+  db.users.update([
+    {$and: [
+      { 'purchased.plan.customerId': { $ne:null } },
```

Comment:
Nitpick here, but could you put a space between the `:` and the value for each of these?